src/App.js

```diff
@@ -1,15 +1,79 @@
 import React, { useState } from 'react';
 import './App.css';
+import posts from './pages/data/posts.json';
+import Login from './pages/login/login.js';
+import Logout from "./pages/logout/logout";
+import BlogPostPage from './pages/blogpostpage/blogpostpage.js';
+import BlogPosts from './pages/blogposts/blogposts.js';
+import {Switch, Route, Redirect, NavLink, } from 'react-router-dom';
 
 function App() {
   // We houden in de state bij of iemand is "ingelogd" (simpele versie)
   const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
+    console.log("blogpost datum" , posts);
 
   return (
-    <div>
-      Maak hier jouw prachtige blog-applicatie!
-    </div>
-  );
+      <>
+         <ul id="balk">
+          <li>
+            <NavLink to="/">Home</NavLink>
+          </li>
+           {isAuthenticated === false &&
+           <li>
+             <NavLink to="/login" activeClassName="activelink">Login</NavLink>
+           </li>
+           } : {isAuthenticated === true &&
```

Comment:
deze 
: 
lijkt mij overbodig, hij staat niet zo netjes onder de Home link in de navigatiebalk op de pagina